xchange-okcoin/src/main/java/org/knowm/xchange/okcoin/OkCoin.java

```diff
@@ -143,4 +144,9 @@ public OKCoinWithdraw withdraw(@FormParam("api_key") String api_key, @FormParam(
       @FormParam("chargefee") String chargefee, @FormParam("trade_pwd") String trade_pwd, @FormParam("withdraw_address") String withdraw_address,
       @FormParam("withdraw_amount") String withdraw_amount) throws IOException;
 
+  @POST
+  @Path("account_records.do")
+  OkCoinAccountRecords getAccountRecords(@FormParam("api_key") String apikey, @FormParam("symbol") String symbol, @FormParam("type") String type,
+                                         @FormParam("current_page") String currentPage, @FormParam("page_length") String pageLength, @FormParam("sign") ParamsDigest sign)
```

Comment:
I would like to address the comment on OkCoinRecords.java (Long to Date) together with this as separate issue. It's not going to be straight forward and I would  require more testing for these changes and I don't want to further delay the delivery of this enhancement if possible. I do hope its ok.